opentracing-impl-java8/src/test/java/io/opentracing/TestSpanImpl.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2016 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing;
+
+public class TestSpanImpl extends AbstractSpan {
```

Comment:
This was basically being inlined in two places already, so I broke it out. Note that this is not part of the core OT-java artifact.